src/NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp

```diff
@@ -296,9 +296,6 @@ struct DerivCompute
  * Computes the spatial derivative for a single tensor represented by
  * 'TensorTag' in the frame mapped to by 'InverseJacobianTag'. It takes a
  * single Tensor designated by 'TensorTag', the inverse Jacobian, and a mesh.
- *
```

Comment:
you squashed this into the wrong commit :)

Reply:
sadness, pain. Will force push the correct commits in a sec